README.md

```diff
@@ -113,13 +114,23 @@ rails server
 
 Open your web browser and go to http://localhost:3000 to access the Blog App.
```

Comment:
### I am requesting these changes here because the relevant files not part of the files changed in the PR.

- [ ] When I load the app in the browser, I should be taken to a splash screen, instead I am taken to the login page. Kindly fix that.

  Your Implementation | Behance design splash screen
  :---:|:---:
  ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/cb0e772b-ee0d-4011-8ed3-699efae9c51f) | ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/2678e2be-b5a5-496f-bc39-b99b0808989f)